docs/lll.rst

```diff
@@ -0,0 +1,21 @@
+###
+LLL
+###
+
+.. _lll:
+
+LLL is a low-level language for the EVM based on s-expressions.
```

Comment:
Maybe: with an s-expressions syntax.

Reply:
Possibly add "(Lisp Like Language")

Reply:
It has like 4-5 acronyms, one other: `Little Lovely Language`.

I'd avoid it in this PR.

Reply:
No problem, its fine as is, just to clarify:

* I don't even bother to assess "Little Lovely"
* "Lisp Like" clarifies things

Reply:
I think the `s-expressions` part clarifies that it is "lisp like", but feel free to open a PR with changes to the LLL chapter.

Reply:
Nah, for me its ok as is! Thanks for the quick processing!